stellarphot/plotting/aij_plots.py

```diff
@@ -43,7 +43,7 @@ def seeing_plot(
     file_name : optional, string
         if entered, file will save as png with this name
 
-    aperture_settings : optional, `stellarphot.settings.ApertureSettings`
+    aperture_settings : optional, `stellarphot.settings.PhotometryApertures`
         The aperture settings used to create the plot. If not provided, the
         aperture radius will be set to 4 * HWHM, the inner annulus will be set
         to radius + 10, and the outer annulus will be set to radius + 25.
```

Comment:
You probably want to update docs to indicate the default FWHM also, and rename the variable from `aperture_settings` (sorry!)